<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_list_with_tablewidget</class>
 <widget class="QMainWindow" name="window_list_with_tablewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Móviles</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/icons/icon_mobile_phone.png</normaloff>images/icons/icon_mobile_phone.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Listar móviles</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>780</width>
      <height>440</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; border: 2px solid navy;</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>id</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Marca</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Modelo</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>SS.OO.</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Memoria</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Oferta</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tecnología</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Foto</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>170</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color:#33bdef;
	border-radius:20px;
	border:3px solid #057fd0;
	color:#ffffff;
	font-family:Verdana;
	font-size:16px;
	font-weight:bold;
</string>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icons/icon_delete.png</normaloff>images/icons/icon_delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>510</y>
      <width>170</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">	background-color:#33bdef;
	border-radius:20px;
	border:3px solid #057fd0;
	color:#ffffff;
	font-family:Verdana;
	font-size:16px;
	font-weight:bold;
</string>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icons/icon_edit.png</normaloff>images/icons/icon_edit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
